tests/tests_automl/test_targets.py

```diff
@@ -34,13 +34,12 @@ def test_bin_class_01(self):
             algorithms=["Xgboost"],
             train_ensemble=False,
             explain_level=0,
+            start_random_models=1,
         )
-        automl.set_advanced(start_random_models=1)
         automl.fit(X, y)
         pred = automl.predict(X)
-        for col in ["prediction_0", "prediction_1", "label"]:
```

Comment:
Maybe there should be `predict_all` method?

Reply:
Implemented